vmdb/app/models/ems_refresh/parsers/openshift.rb

```diff
@@ -14,17 +14,17 @@ def get_routes(inventory)
       end
 
       def get_projects(inventory)
-        process_collection(inventory["project"], :container_projects) { |n| parse_project(n) }
+        process_collection(inventory["project"].each, :container_projects) { |item| parse_project(item) }
```

Comment:
You haven't followed my suggestion here of just using:

    inventory["project"].each { |item| parse_project(item) }

I think that's why you're collecting the items twice.